zerver/lib/bugdown/fenced_code.py

```diff
@@ -239,6 +270,28 @@ def done(self):
             output.append('')
         return output
 
+    def format_quote(self, text):
+        # type: (Text) -> Text
+        paragraphs = text.split("\n\n")
+        quoted_paragraphs = []
+        for paragraph in paragraphs:
+            lines = paragraph.split("\n")
+            quoted_paragraphs.append("\n".join("> " + line for line in lines if line != ''))
+        return "\n\n".join(quoted_paragraphs)
```

Comment:
I did it to keep the `format_xxx` functions in the same order as the `Handler` classes above.

To keep it alphabetical, now I've moved the handler and restored `format_quote`'s original position. Does that look better?